packages/client/src/game-core/input/InputHandler.ts

```diff
@@ -35,11 +64,17 @@ export class InputHandler {
     }
 
     isDown(key: string) {
-        return this.pressedKeys[key.toUpperCase()]
+        return (
+            this.pressedKeys[key.toUpperCase()] ||
+            this.gamepadButtons[key.toUpperCase()]
+        )
     }
 
     isUp(key: string) {
-        return !this.pressedKeys[key.toUpperCase()]
+        return (
+            !this.pressedKeys[key.toUpperCase()] ||
```

Comment:
Это условие ломает двойной прыжок. Возможно стоит сделать так. Нужно проверить на геймпаде, но двойной прыжок точно не сломает
```suggestion
            !this.pressedKeys[key.toUpperCase()] &&
```